### DISCUSSION TITLE
Example of best practices in production?

### DISCUSSION THREAD

**Discussion by jeremy303** [sequelize/umzug#84] [Q&A]
I would love to see an example of current best practices when using Umzug for Sequelize migrations in a production system.

Can someone point me to a good example?

Thanks.

**Comment by jukkah** [comment]
Sorry for delayed answer.

We don't have any real world examples right now. For now, if you look around the internet for projects using Sequelize you might find some example migrations too. In this or next year, I plan to play a lot with Umzug migrations in my lab. But it wont help you right now.

**Comment by jukkah** [comment]
In v2.0 I'll add some usage examples.
